File: HackerRank Solutions/Mathematics/Fundamentals/Minimum Height Triangle/Minimum Height Triangle.c
#include <math.h>
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <assert.h>
#include <limits.h>
#include <stdbool.h>

int lowestTriangle(int base, int area){
    // Complete this function
    return(ceil((float)(2 * area)/base));
}

int main() {
    int base; 
    int area; 
    scanf("%d %d", &base, &area);
    int height = lowestTriangle(base, area);
    printf("%d\n", height);
    return 0;
}

File: HackerRank Solutions/Mathematics/Fundamentals/Minimum Height Triangle.py
#!/bin/python3

import math

def lowestTriangle(base, area):
    # Complete this function
    return math.ceil(2 * area / base)
base, area = input().strip().split(' ')
base, area = [int(base), int(area)]
height = lowestTriangle(base, area)
print(height)